_data/navbar.yml

```diff
@@ -1,9 +1,11 @@
-- title: Hlavní stránka
-  url: /
+# - title: Hlavní stránka
+#   url: /
```

Comment:
Co je praktický dopad téhle změny? Znamená to že se z podstránek nepůjde dostat na hlavní stránku, nebo máme pořád cestu přes nějaké proklikávací logo apod?

Reply:
v rohu navbaru je logo, který tě vezme na hlavní stránku